src/Sarif.Multitool/Program.cs

```diff
@@ -46,11 +46,11 @@ public static int Main(string[] args)
                 (ExportValidationConfigurationOptions options) => new ExportValidationConfigurationCommand().Run(options),
                 (ExportValidationRulesMetadataOptions options) => new ExportValidationRulesMetadataCommand().Run(options),
                 (FileWorkItemsOptions fileWorkItemsOptions) => new FileWorkItemsCommand().Run(fileWorkItemsOptions),
-                (ResultMatchingOptions baselineOptions) => new ResultMatchingCommand().Run(baselineOptions),
                 (MergeOptions mergeOptions) => new MergeCommand().Run(mergeOptions),
                 (PageOptions pageOptions) => new PageCommand().Run(pageOptions),
                 (QueryOptions queryOptions) => new QueryCommand().Run(queryOptions),
                 (RebaseUriOptions rebaseOptions) => new RebaseUriCommand().Run(rebaseOptions),
+                (ResultMatchingOptions baselineOptions) => new ResultMatchingCommand().Run(baselineOptions),
```

Comment:
>ResultMatchingCommand [](start = 63, length = 21)

I didn't set in the alphabetical order because the resultmatching is equal match-results-foward command.

Reply:
Would it be worthwhile to rename the file/class to something like "MatchResultOptions" for consistency?

---
In reply to: [561433654](https://github.com/microsoft/sarif-sdk/pull/2252#discussion_r561433654) [](ancestors = 561433654)

Reply:
Let's do it in another pr, separating what we are doing right now :)

---
In reply to: [561480791](https://github.com/microsoft/sarif-sdk/pull/2252#discussion_r561480791) [](ancestors = 561480791,561433654)

Reply:
Done.

---
In reply to: [564739449](https://github.com/microsoft/sarif-sdk/pull/2252#discussion_r564739449) [](ancestors = 564739449,561480791,561433654)